tbdex/sign.go

```diff
@@ -26,3 +30,39 @@ func Sign(digester Digester, bearerDID did.BearerDID) (string, error) {
 
 	return signature, nil
 }
+
+// Digest generates a SHA-256 hash of the canonicalized input payload.
+func Digest(payload interface{}) ([]byte, error) {
```

Comment:
can use `any` instead of `interface{}`. Also maybe a better name for this function is `DigestJSON` because you're json marshaling and using JCS to canconicalize